lib/fs.js

```diff
@@ -41,6 +41,24 @@ const isWindows = process.platform === 'win32';
 const DEBUG = process.env.NODE_DEBUG && /fs/.test(process.env.NODE_DEBUG);
 const errnoException = util._errnoException;
 
+function getOptions(options, defaultOptions) {
+  if (options == null || typeof options === 'function') {
```

Comment:
Sometimes people pass `null` and sometimes `undefined`. That is why I wrote it like this. Eg: https://github.com/isaacs/node-graceful-fs/blob/master/graceful-fs.js#L89